readme.md

```diff
@@ -249,48 +245,16 @@ func main() {
 
 ```
 
-### Shipping multiple files to Pinata via the rainbow bridge
-
-```go
-// Upload multiple files
-uploadedFiles, err := bridge.UploadMultiFile(bifrost.MultiFile{
-	Files: []bifrost.File{
-		{
-			Path:     "../shared/image/hair.jpg",
-			Filename: "hair_of_opensaucerer.jpg",
-			Options: map[string]interface{}{
-				bifrost.OptMetadata: map[string]string{
-					"originalname": "hair.jpg",
-				},
-			},
-		},
-		{
-			Path:     "../shared/image/bifrost.webp",
-			Filename: "bifrost_bridge.webp",
-			Options: map[string]interface{}{
-				bifrost.OptMetadata: map[string]string{
-					"originalname": "bifrost.jpg",
-					"universe":     "Marvel",
-				},
-			},
-		},
-	},
-
-	GlobalOptions: map[string]interface{}{
-		bifrost.OptPinata: map[string]interface{}{
-			"cidVersion": 1,
-		},
-	},
-})
-if err != nil {
-	fmt.Println(err)
-	return
-}
-
-for _, file := range uploadedFiles {
-	fmt.Printf("Uploaded file: %s to %s\n", file.Name, file.Preview)
-}
+# Installation
+To install the Bifrost package, run the following command in your terminal:
+```bash
+go get github.com/opensaucerer/bifrost
 ```
+# Usage
+If you want to learn more about how Bifrost is creating different methods to make it easier to use different cloud providers, you can follow these links: 
+- [Google Cloud Storage (GCS)](gcs\doc.md)
+- [Amazon S3](s3\doc.md)
+- [Pinata](pinata\doc.md)
```

Comment:
`Pinata Cloud`

Reply:
I've resolved the requested changes and it's ready for review.